### PR TITLE
update bom to the latest one

### PR DESCRIPTION
**Pull request by reuvenlax**

### CODE REVIEW

**Comment by reuvenlax** [comment]
run dataflow validatesrunner

**Comment by reuvenlax** [comment]
Run Java PreCommit

**Comment by reuvenlax** [comment]
Run Java PreCommit

**Comment by reuvenlax** [comment]
Run Java PreCommit

**Comment by reuvenlax** [comment]
run dataflow validatesrunner

**Comment by reuvenlax** [comment]
Run Python PreCommit

**Comment by reuvenlax** [comment]
Run Portable_Python PreCommit

**Comment by reuvenlax** [comment]
Run Java PreCommit

**Comment by reuvenlax** [comment]
friendly ping

File: buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy [hunk 1]
```diff
@@ -609,7 +609,7 @@ class BeamModulePlugin implements Plugin<Project> {
         // The GCP Libraries BOM dashboard shows the versions set by the BOM:
         // https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/26.1.3/artifact_details.html
```
Comment (kileys) [inline buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:610]:
```suggestion
        // https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/26.1.5/artifact_details.html
```

**Review by kileys — APPROVED** [review]

**Comment by kileys** [comment]
Run Java_Amazon-Web-Services2_IO_Direct PreCommit

**Comment by reuvenlax** [comment]
Run Java PreCommit

**Comment by reuvenlax** [comment]
Run Java PreCommit

**Pull request merged**